src/universal/types/App.types.ts

```diff
@@ -92,6 +92,7 @@ export interface StatusLineItem {
   datePublished: string;
   description?: string;
   documents?: GenericDocument[];
+  resultaat?: string;
```

Comment:
Waarom heb je dit toegevoegd?

Reply:
omdat je die moet laten zien op de thema pagina toch?

Reply:
Het is een line-item type (step) Deze wordt gebruikt voor de statustrein.